src/DataManagement/EntryList.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using ePiggyWeb.Utilities;
+
+namespace ePiggyWeb.DataManagement
+{
+    public class EntryList : List<Entry>
```

Comment:
Why should this be used instead of simple Lists of entries? Just curious

Reply:
There were a few reasons on why I implemented this change:
- First this class contains EntryType variable, which saves a lot of code later on when calling methods and so on.
- If we ever need to add more specific requirements for a list containing Entries, this will make it easier not to have to create a new class and amend the code everywhere. 
- We can create override methods if we need to for the List class
- Debatable, but could improve readability and ease of writing code, instead of always writing List<Entry> we can just write EntryList
- I think that this might be useful later to implement Dependency Injection and other more difficult requirements